meta-xt-dom0/recipes-kernel/linux/linux-generic-armv8.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-BRANCH = "v5.10.41/rcar-5.1.4.1-xt0.1"
+BRANCH = "v5.10.41/rcar-5.1.4.1-xt0.2"
```

Comment:
Didn't product layer and/or YAML file override this?

Reply:
In particular meta-xt-prod-devel-rcar didn't touch this. In order to change the Linux branch for Dom0/DomD/DomU I make a change here:
meta-xt-common/meta-xt-dom0/recipes-kernel/linux/linux-generic-armv8.bbappend
meta-xt-rcar/meta-xt-rcar-driver-domain/recipes-kernel/linux/linux-renesas_5.10.bbappend
meta-xt-rcar/meta-xt-rcar-domu/recipes-kernel/linux/linux-renesas_%.bbappend

I know that Dom0 Linux ought to be generic one, but in order to simplify things (having the single branch for all domains, etc) we use the same BSP based branch although with tiny defconfig.

Reply:
Yeah, this is the problem, as, again, this is `meta-xt-common` but we are using Renesas linux branch for some reason (BTW, I had issues when Dom0 kernel tried to initialize renesas peripherals). 

Okay, let's leave it as is for a time being, but I am asking you to create issue about this.

Reply:
"+"